app/serializers/highlight_serializer.rb

```diff
@@ -0,0 +1,3 @@
+class HighlightSerializer < ActiveModel::Serializer
+  attributes :id, :source_type, :user_uuid, :source_id, :location_strategies
+end
```

Comment:
Unused?

Reply:
this is used to serialize the highlight in the #create response body

Reply:
removed in leu of using the binding